Packs/CommonScripts/pack_metadata.json

```diff
@@ -47,6 +47,14 @@
         "RemoteAccess": {
             "mandatory": false,
             "display_name": "RemoteAccess v2"
+        },
+        "rasterize": {
```

Comment:
Why did we add these dependencies? 
It shouldn't be mandatory: true

Reply:
the script DBotPredictURLPhishing.py uses rasterize and Whois